amazon-kinesis-client-multilang/src/test/java/software/amazon/kinesis/multilang/MultiLangDaemonConfigTest.java

```diff
@@ -200,4 +198,9 @@ public void testPropertyValidation() {
         }
     }
 
+    @Test
+    public void testActualPropertiesFile() throws Exception {
+        new MultiLangDaemonConfig(FILENAME);
```

Comment:
> What is this testing? Just constructor works fine?

I added Javadoc. Snippet: "catch any issues which might arise if there is a discrepancy between reality and mocking".

> Maybe we want to assert constructed instance is not null?

`assertNotNull(...)` gets marked as redundant because the construction is guaranteed non-null iff an Exception is not thrown.

Reply:
But if you dont assert and the test throws exception it may not show as failed, isnt it?

Reply:
I think it does mark the test failed, so this is okay.